src/state/apis/zapper/zapperApi.ts

```diff
@@ -418,8 +433,15 @@ export const zapper = createApi({
                   // This is our best bet until we bring in the concept of an "DefiType.GenericOpportunity"
                   const defiType = DefiType.Staking
 
-                  // Actually defined because we pass networks in the query params
-                  const assetId = zapperAssetToMaybeAssetId(asset)
+                  const topLevelAsset = (() => {
+                    const maybeLpAsset = asset.tokens.find(
+                      token => token.metaType === 'supplied' || token.metaType === 'borrowed',
+                    )
+                    if (maybeLpAsset) return maybeLpAsset
+                    return asset
+                  })()
+
+                  const assetId = zapperAssetToMaybeAssetId(topLevelAsset)
```

Comment:
The second guts of this, the top-level asset was incorrectly assumed previously resulting in *slightly* wrong fiat values

<img width="1304" alt="image" src="https://github.com/shapeshift/web/assets/17035424/5c1da03f-9bc5-4e02-bd35-ec606a75b97a">